official/nlp/keras_nlp/layers/transformer_encoder_block.py

```diff
@@ -20,12 +20,10 @@
 @tf.keras.utils.register_keras_serializable(package="keras_nlp")
 class TransformerEncoderBlock(tf.keras.layers.Layer):
   """TransformerEncoderBlock layer.
-
```

Comment:
Please keep these empty lines. This is part of our coding style.

Reply:
I think it is done, my IDE trolled me. Let me know if it is ok now.